rosflight/src/rosflight_io.cpp

```diff
@@ -707,8 +712,28 @@ void rosflightIO::handle_version_msg(const mavlink_message_t &msg)
     version_pub_ = nh_.advertise<std_msgs::String>("version", 1, true);
   }
   version_pub_.publish(version_msg);
+#ifdef GIT_VERSION_STRING // Macro so that is compiles even if git is not available
+  std::string git_version_string = GIT_VERSION_STRING;
+  std::string rosflight_major_minor_version =
+      git_version_string.substr(0, git_version_string.find('.', git_version_string.find('.') + 1));
+  std::string firmware_version(version.version);
+  std::string firmware_major_minor_version =
+      firmware_version.substr(1, firmware_version.find('.', firmware_version.find('.') + 1) - 1);
```

Comment:
is there any way we could abstract this?  (It's a little hard to grok this as-is)
```
int get_major_version(const std::string& version);
int get_minor_version(const std::string& version);
```
or whatever